package.json

```diff
@@ -8,7 +8,9 @@
   "dependencies": {
     "css-layout": "^1.1.0",
     "d3": "^3.5.4",
+    "d3-path": "^0.1.5",
```

Comment:
**d3fc-path** only depends on **d3-path** as a dev dependency (used for testing). **d3fc-path** just takes in the context you give it, which from the **d3fc/svg** stuff is **d3-path**.
Wow that's a lot of paths...